elm-review confused about directories

Hello!
## Versions
```
$ elm-review --version
2.4.6
```

## How I called elm-review
```
$ elm-review --config review --elmjson client/_share/elm.json client/_share/src/Data/Labels.elm
```

## Directory structure
```
/
├── elm.json
├── review/ (should be all good since normal `elm-review` runs work, skipping)
├── client/
│   ├── _share
│   │   ├── elm.json
│   │   ├── src/
│   │   │   ├── Data/
│   │   │   │   ├── Labels.elm
```

## Error encountered
```
Error: Cannot find module 'client/_share/elm-stuff/generated-code/jfmengels/elm-review/cli/2.4.6/elm-parser/elm-syntax-v7.2.1.js'
Require stack:
- /home/martin/Localhost/gwi/pro-next/node_modules/elm-review/lib/parse-elm-worker.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:923:15)
    at Function.Module._load (node:internal/modules/cjs/loader:768:27)
    at Module.require (node:internal/modules/cjs/loader:995:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at workerParseElm (/home/martin/Localhost/gwi/pro-next/node_modules/elm-review/lib/parse-elm-worker.js:18:23)
    at MessagePort.<anonymous> (/home/martin/Localhost/gwi/pro-next/node_modules/elm-review/lib/parse-elm-worker.js:11:3)
    at MessagePort.[nodejs.internal.kHybridDispatch] (node:internal/event_target:459:20)
    at MessagePort.exports.emitMessage (node:internal/per_context/messageport:18:26)
```

Note `elm-stuff/generated-code/jfmengels/elm-review/...` does exist in the top-level `elm-stuff`, but not in the `client/_share/elm-stuff`.

--- a/lib/options.js
+++ b/lib/options.js
@@ -181,7 +181,8 @@ path.relative(process.cwd(), 'elm.json')
     elmModulePath: (appHash /* : string */) =>
       path.join(elmStuffFolder(), 'review-applications', `${appHash}.js`),
     elmParserPath: (elmSyntaxVersion /* : string */) =>
-      path.join(
+      path.resolve(
+        process.cwd(),
         elmStuffFolder(),
         'elm-parser',
         `elm-syntax-v${elmSyntaxVersion}${args.debug ? '-debug' : ''}.js`

